environments/skeleton/{{cookiecutter.environment}}/tofu/compute.tf

```diff
@@ -21,6 +21,7 @@ module "compute" {
   extra_volumes = lookup(each.value, "extra_volumes", {})
 
   compute_init_enable = lookup(each.value, "compute_init_enable", [])
+  ignore_image_changes = lookup(each.value, "ignore_image_changes", false)
```

Comment:
This needs to be added to the docs for the `compute` variable in `skeleton/{{cookiecutter ...}}/tofu/variables.tf`